skytemple_randomizer/frontend/gtk/ui_util.py

```diff
@@ -79,6 +79,7 @@ def show_about_dialog(parent: Gtk.Widget):
 Adex (via patches + JP support) https://github.com/Adex-8x
 Darkaim (JP support)
 Laioxy (JP support via pmdsky-debug) https://github.com/Laioxy
+in2erval (via patches + JP support) https://github.com/in2erval
```

Comment:
"Patches" usually refers to ASM patches, while "JP support" normally means making changes to the code to ensure the randomizer is compatible with JP ROMs.
This change is not an ASM patch, and it applies to all versions of the game, so I'm not sure that's the best description.

It might be better to simply list the name, or maybe "(code contributions)". Not sure what would work better here.

Reply:
I see, in that case I can change it to "code contributions"